charts/ratify/templates/inline-certificate-provider.yaml

```diff
@@ -1,10 +1,10 @@
-{{- if eq .Values.akvCertConfig.enabled false }}  
+{{- if .Values.notaryCert}} 
```

Comment:
small nit: extra space after `notaryCert`. Not a block though.

Reply:
done

Reply:
I think the change got overriden?

Reply:
Looks like it juggling between ui and cli for commits and squashes.. shud be fixed now :)